src/libs/actions/IOU.js

```diff
@@ -99,26 +123,166 @@ function startLoadingAndResetError() {
 }
 
 /**
- * Creates IOUSplit Transaction
+ * Request money from another user
  *
- * @param {Object} params
- * @param {Number} params.amount
- * @param {String} params.comment
- * @param {String} params.currency
- * @param {String} params.debtorEmail
+ * @param {Object} report
+ * @param {Number} amount
+ * @param {String} currency
+ * @param {String} recipientEmail
+ * @param {String} debtorEmail
+ * @param {String} comment
  */
-function createIOUTransaction(params) {
-    startLoadingAndResetError();
-    DeprecatedAPI.CreateIOUTransaction(params)
-        .then((response) => {
-            if (response.jsonCode !== 200) {
-                processIOUErrorResponse(response);
-                return;
-            }
+function requestMoney(report, amount, currency, recipientEmail, debtorEmail, comment) {
+    let chatReport = lodashGet(report, 'reportID', null) ? report : null;
+    let isNewChat = false;
+    if (!chatReport) {
+        chatReport = ReportUtils.getChatByParticipants([debtorEmail]);
+    }
+    if (!chatReport) {
+        chatReport = ReportUtils.buildOptimisticChatReport([debtorEmail]);
+        isNewChat = true;
+    }
+    let iouReport;
+    if (chatReport.hasOutstandingIOU) {
+        iouReport = iouReports[`${ONYXKEYS.COLLECTION.REPORT_IOUS}${chatReport.iouReportID}`];
+        iouReport.total += amount;
```

Comment:
@mountiny I think that [this comment ](https://github.com/Expensify/App/pull/11597#discussion_r1002127005) applies in this case as well right? We need to potentially update the new owners and totals too.

Reply:
@Gonals oh that is correct, lets say you request money from someone you owe to already and you request more, we need to swap the sign. 

Lets reuse the method which Carlos put into the IOUUtils.